blog/_posts/Hola, Amigos.md

```diff
@@ -0,0 +1,118 @@
+
+I am also a sports enthusiast, with basketball being one of my favourite sports. A book in hand, headphones and a decent internet connection is all I need when in melancholy. Quora comes to the rescue when the critic in me tends to rise.
+
+Elevating Medical Sciences using computers and technology is my passion, and it will be my future. I want to become a Computer Scientist and be known for my talent, and not because I’m a *woman*. I want people to understand that *I am more than what they can see*."
+
+I have had this habit of imaginng things in my head, things that I aspire to do and achieve, but the results are not always the way I expected them to be. But then, I take a deep breath and reflect back, and am reminded of the famous quote from the *Harry Potter series*
+
+> *“Of course it is happening inside your head, Harry, but why on earth should that mean that it is not real?” ― Albus Dumbledore, Harry Potter and the Deathly Hallows*
+
+But this is just a way to pacify myself and keep dreaming with a head held high. :D
+
+## Our Team
+
+### Our mentor:
+[**Michael Ferguson**](https://github.com/mppf) and [**Ben Albrecht**](https://github.com/ben-albrecht):
+
+We are delighted to have them as our mentors for the project, and hope that under their able guidance, we can have a steep learning curve. Also, we are grateful to them for helping us during the application process.
+
+### Our Official coaches:
+We are grateful to them for agreeing to coach us, guide us, and help us grow and learn, as we embark upon this journey. Again, we are extremely thankful to him for all the mentoring at the time of applications.
+
+ - [**Louis Jenkins**](https://louisjenkinscs.github.io/): He is a  recent Computer Science graduate from Bloomsburg interests and experience in the low-level areas such as compilers, operating systems, and the design and implementation of high-performance data structures. [Source](https://louisjenkinscs.github.io/)
+
+- [**Sarthak Munshi**](https://saru.science/about/): He is currently, working at HackerRank. Most of his work is open-sourced and can be found on his [github profile](https://github.com/saru95). [Source](https://saru.science/about/)
+
+- [**Engin Kayraklioglu**](https://github.com/e-kayrakli)
+
+### Volunteer Coaches: 
+They are willing to help us at any point of time during the project and we are thankful to them for that. 
+
+- [**Kushal Singh**](https://in.linkedin.com/in/kushalsingh007): He is currently working as a platform engineer at Direct-I, India, and has previously been part of Google Summer Of Code. [Source](https://in.linkedin.com/in/kushalsingh007)
+
+- [**Lydia Duncan**](https://github.com/lydia-duncan)
```

Comment:
It looks weird to have small introduction text for everyone except this person. Please either add for all or none.